tools/debug.sh

```diff
@@ -24,8 +24,13 @@ function main()
     launcher_text_addr=$(get_text_addr "$launcher_path")
     local gdbinit="$script_dir/gdbinit"
 
-    cat >/tmp/x.gdb<<EOF
+    local arch=$(uname -a)
```

Comment:
Please use the following pattern:

```console
local arch
arch=$(uname -a)
```

From `shellcheck`:

```
    local arch=$(uname -a)
          ^-- SC2155: Declare and assign separately to avoid masking return values.
```

Reply:
Done